src/nvim/main.c

```diff
@@ -2062,34 +2033,28 @@ static int file_owned(char *fname)
 }
 #endif
 
-/*
- * Give an error message main_errors["n"] and exit.
- */
-static void
-mainerr (
-    int n,                  /* one of the ME_ defines */
-    char_u *str       /* extra argument or NULL */
-)
+/// Prints the following then exits:
+/// - Nvim's version
```

Comment:
I forgot to update the comment (I'll do so), but I did that on purpose. From the commit message:
```
Don't print version when called. mainerr() only handles errors
related to command line arguments, so the version isn't much help at
all.
```
and the mentioned errors:
```
Unknown option argument
Too many edit arguments
Argument missing after
Garbage after option argument
Too many "+command", "-c command" or "--cmd command" arguments
```